image-filters/image-filters.markdown

```diff
@@ -573,56 +570,56 @@ of colors in a program is very important to understanding how our filters work.
 To prepare ourselves for working on our next filter, let's explore the concept
 of color a bit more.
 
-You may not have known it at the time, but we were unwittingly using a concept
-in the previous section called a "color space", which is way of representing
-color digitally. Kids mixing paints learn that all colors can be made from
-other colors, but things work slightly differently in digital (less risk of
+We were using a concept
+in the previous section called "color space", which is way of representing
+color digitally. Kids mixing paints learn that colors can be made from
+other colors; things work slightly differently in digital (less risk of
 being covered in paint!) but similar. Processing makes it really easy to work
```

Comment:
I left it the way it is because it's conversational. If we wanted to be very proper I would remove "but similar" altogether, because it doesn't add much to "slightly differently". I think it's fine the way it is.